.gitignore

```diff
@@ -38,6 +38,7 @@ cscope.*
 **/pyformat
 SOURCE_VERSION
 *.s??
+!go.sum
```

Comment:
from `git check-ignore` it would seem its the `*.s??` line that is causing go.sum to be excluded - trying to understand why it was added

Reply:
k - added #28282 to address this issue

Reply:
#28282 just landed so this should not be necessary now